graphql/src/main/java/com/linecorp/armeria/common/graphql/scalar/MultipartFileScalar.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2022 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package com.linecorp.armeria.common.graphql.scalar;
+
+import com.linecorp.armeria.common.multipart.MultipartFile;
+
+import graphql.schema.Coercing;
+import graphql.schema.CoercingParseLiteralException;
+import graphql.schema.CoercingParseValueException;
+import graphql.schema.CoercingSerializeException;
+import graphql.schema.GraphQLScalarType;
+
+/**
+ * A multipart-file scalar that converts a file path string into a
+ * {@link com.linecorp.armeria.common.multipart.MultipartFile}.
+ */
+final class MultipartFileScalar {
```

Comment:
How about making enum singleton?

Reply:
Oops, could you revert this change?

I thought `MultipartFileScalar.INSTANCE` is `MultipartFileScalar` before the code change, but it was `GraphQLScalarType` so we shouldn't make this enum singleton. 😅 

So how about reverting and renaming `INSTANCE` field to `scalarType`?

Reply:
> So how about reverting and renaming INSTANCE field to scalarType?

Isn't this a bit odd because it's `static` ? 🤔

Reply:
Do you mean it's weird to use camel case?
Conventionally, we need to use camel case for a final reference. See https://stackoverflow.com/questions/1417190/should-a-static-final-logger-be-declared-in-upper-case

One exception is that we use `INSTANCE` for a singleton field of the class. However, I don't think it's the case because `MultipartFileScalar` and `GraphQLScalarType` are different types. 😄

Reply:
Oh... Thanks for the explanation 🤣